.github/workflows/docker_build_publish.yml

```
@@ -1,9 +1,13 @@
-name: Build and push master and develop container
+name: Build and publish develop containers
 on:
   push:
     branches:
-      - master
-      - develop
+      - "develop"
+  pull_request:
+    paths:
+      - "BALSAMIC/containers/**"
+    branches-ignore:
+      - "release_*"
 jobs:
   main:
     name: Docker image build push
```

```suggestion
    name: Docker image build and publish to develop
```
Do we want to be more specific here?

---

I would say yes, but what we include here appear later in the PR checks. A long name makes it difficult to access/read. But I agree with you, I can add some docstrings, that's a good idea 👍